components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIManagerConfiguration.java

```diff
@@ -624,6 +633,8 @@ private void readChildElements(OMElement serverConfig,
                     jsonObject.put(APIConstants.CustomPropertyAttributes.REQUIRED, isRequired);
                     customProperties.add(jsonObject);
                 }
+            } else if (APIConstants.MARKETPLACE_ASSISTANT.equals(localName)) {
```

Comment:
From where this localName comes from? if it is a user provided value, lets do a equalIgnoreCase check

Reply:
Similar equal checks are done on localName within the same scope.